server/schemas/typeDefs.js

```diff
@@ -86,8 +86,13 @@ const typeDefs = gql`
   type Mutation {
     addUser(newUser: NewUserInput!): Auth
     login(email: String!, password: String!): Auth
-    createJob(newJob: NewJobInput!): Job
-    createProposal(newProposal: NewProposalInput!): Proposal
+    createJob(newJob: JobInput!): Job
+    createProposal(newProposal: ProposalInput!): Proposal
+    deleteJob(jobId: ID!): Job
+    deleteProposal(proposalId: ID!): Proposal
+    editJob(updateJob: JobInput!): Job
```

Comment:
for editJob and editProposal we want to pass in the jobID and proposalID as well to use as the filter in the resolvers